packages/client/src/client.js

```diff
@@ -2,7 +2,7 @@ import fs from 'fs';
 import PercyEnv from '@percy/env';
 import { git } from '@percy/env/utils';
 import logger from '@percy/logger';
-import { waitForTimeout } from '../../core/src/utils.js';
+import { waitForTimeout } from '../../core/dist/utils.js';
```

Comment:
this is not correct way - you should add `@percy/core` as dependency in `@percy/client` and require it.

Reply:
got it thanks :)